src/stores/PreferenceStore/PreferenceStore.ts

```diff
@@ -703,25 +728,24 @@ export class PreferenceStore {
 
```

Comment:
Would be nice if we can add TSDoc for `setPreference`, stating that enum values (instead of a number list or a binary number) are required for log events and wcs matching.

Reply:
I tried to add some TSDoc in PereneceStore.